### PR TITLE
VAGOV-6192

### PR DESCRIPTION
**Pull request by ethanteague**
Details: https://va-gov.atlassian.net/browse/VAGOV-6192
@kevwalsh I've created the new listing types, and config'd perms, updated tests, etc. Would be great if you could have a look at my work up to this point.

### CODE REVIEW

**Comment by kevwalsh** [comment]
@ethanteague this is huge! 

I'm going to note some stuff here:
- [ ] For node forms: The right sidebar fieldgroup order should be Governance first, then Meta tags, menu settings, then URL Alias (not a big deal, but quick fix for consistency).
- [ ] Editorial workflow: Each of these new content types should be added to the "Content types" list on admin/config/workflow/workflows/manage/editorial?destination=/admin/config/workflow/workflows. 
  - [ ]The editorial workflow widget should be placed at the top of each node view page too
- [ ] Pathauto config: /[field_office]/[events/stories/news-releases/health-services/locations]
- [ ] Permissions: i think the Create node permission should only be for content_admin, for now. Then everyone gets Edit permission.  i don't think anyone needs any of the new node_override permissions for these content types.

**Comment by kevwalsh** [comment]
@ethanteague i found one more issue. "Press release listing" should be called "News release listing".  "Press release" got renamed "News release" about 6 months ago. It's fine if machine names keep their old press_release versions.

File: config/sync/pathauto.pattern.health_services_listings.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+uuid: ea879b6b-3c90-45e1-baad-0573082de64b
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: health_services_listings
+label: 'Health services listings'
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/health-services/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern.health_services_listings.yml]: this one doesn't need the `/[node:title]`

File: config/sync/pathauto.pattern.locations_listings.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+uuid: b3330ad3-48cd-4e45-a49c-ae2940326e24
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: locations_listings
+label: 'Locations listings'
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/locations/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern.locations_listings.yml]: this one doesn't need the `/[node:title]`

File: config/sync/pathauto.pattern.story_listings.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+uuid: 8567c305-a3ca-4bc4-be93-25479de2f32d
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: story_listings
+label: 'Story listings'
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/stories/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern.story_listings.yml]: this one doesn't need the `/[node:title]`

File: config/sync/pathauto.pattern.news_releases_listings.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+uuid: 682f6678-af5f-4417-b5cc-8a1c8544187e
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: news_releases_listings
+label: 'News releases listings'
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/news-releases/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern.news_releases_listings.yml]: this one doesn't need the `/[node:title]`

File: config/sync/pathauto.pattern._node_field_office_entity_url_path_events.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+uuid: 53739605-8888-4f06-8cec-1bdf9a1eb930
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: _node_field_office_entity_url_path_events
+label: '[node:field_office:entity:url:path]/events'
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/events/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern._node_field_office_entity_url_path_events.yml]: this one doesn't need the `/[node:title]`

File: config/sync/pathauto.pattern.event.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+uuid: fed3baab-17f6-4a11-9170-cbb9c614e603
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: event
+label: Events
+type: 'canonical_entities:node'
+pattern: '[node:field_office:entity:url:path]/events/[node:title]'
```
Comment (kevwalsh) [inline config/sync/pathauto.pattern.event.yml]: this one doesn't need the `/[node:title]`

**Comment by swirtSJW** [comment]
Looks like test still be angry.

**Pull request merged**